packages/aws-cdk/README.md

```diff
@@ -74,7 +74,7 @@ $ cdk init lib --language=typescript
 
 ### `cdk list`
 
-Lists the stacks modeled in the CDK app.
+Lists the stacks and their dependencies modeled in the CDK app.
 
 ```console
 $ # List all stacks in the CDK app 'node bin/main.js'
```

Comment:
We need to update these examples once CLI team decides on the UX for this command.